completion/available/system.completion.bash

```diff
@@ -3,26 +3,23 @@
 # Loads the system's Bash completion modules.
 # If Homebrew is installed (OS X), its Bash completion modules are loaded.
 
-if [ -f /etc/bash_completion ]; then
-  . /etc/bash_completion
+if [[ -r /etc/bash_completion ]] ; then
```

Comment:
I think this one needs to go as well, as it does the same thing as the next `if`, similar to how it was/is for the Homebrew case...

Reply:
@nwinkler I was hoping to get rid of this too, but unfortunately this one needs to stay, as it's the standard wrapper for Debian's bash-completion package. You can test quick with docker:

```
docker run --rm -it debian:stable bash
```
```
apt-get update
apt-get install bash-completion
cat /etc/bash_completion
```

Reply:
Ah, right, got it. Can we maybe turn it into an `elif` (in line 12) so that only one of them gets executed? At the moment, it's loading things twice if you have the completion script in both places...